src/js/options.js

```diff
@@ -100,20 +100,24 @@ document.addEventListener("DOMContentLoaded", () => {
 
 
 // Change settings with the options menu.
-Object.entries(SETTINGS_LIST).forEach(([key, { eventType }]) => {
+Object.entries(SETTINGS_LIST).forEach(([key, { eventType, values }]) => {
   const settingElements = Array.from(document.getElementsByClassName(key));
   settingElements.forEach(button => button.addEventListener(eventType, async e => {
 
-    // Toggle on click: new value is opposite of old value.
-    const value = !(String(HTML.getAttribute(key)).toLowerCase() === "true");
+    // Cycle on click: new value is opposite of old value, or if values is present, new value is next to the old value.
+    let value = !(String(HTML.getAttribute(key)).toLowerCase() === "true");
+    if (values) {
```

Comment:
If `values` is present, don't switch between `true` and `false`. Instead cycle between the values present in `values`.